src/main/resources/data/minecolonies/researches/civilian/4_livesaver2.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "livesaver2",
+  "name": "Livesaver II",
```

Comment:
Shouldn't this be Lifesaver vs Livesaver?

Reply:
Ah, thanks for catching that.

Reply:
Also for Lifesaver I

Reply:
Also for Lifesaver I

Reply:
Thanks.  Got them, and added update handlers to the file loader for both.